tests/ms2/processes/process-list.page.ts

```diff
@@ -165,20 +168,42 @@ export class ProcessListPage {
         await page.waitForURL('**/processes');
       }
 
+      /* Ensure nothing is selected (esc) */
+      await page.getByRole('main').click();
+      await page.getByRole('main').press('Escape');
+      /* Ensure that the search input is cleared */
+      const inputSearch = await page.locator('.ant-input-affix-wrapper');
+      await inputSearch.getByPlaceholder(/search/i).fill('');
+
```

Comment:
When testing with moving into a folder, a check for being in the root folder before deleting could be beneficial, however, then the deletion loop needs to keep track of the processes to delete (deleting a folder deletes everything within it, therefore all ids within it should be removed aswell).